scripts/lib/upgrade-zulip-stage-2

```diff
@@ -430,6 +430,14 @@ if not args.skip_puppet and IS_SERVER_UP:
         sys.exit(1)
 
 
+def run_hooks(path: str) -> None:
+    if not os.path.exists(path):
+        return
+    for script_name in sorted(os.listdir(path)):
```

Comment:
We may want to do something a bit stricter here to avoid Emacs backup files and the like from being run as normal hooks; I feel like that's always a problem with `/etc/` stuff.

Reply:
Since it's arbitrary executable, it's hard to specify anything about the file extension.  We could require `*.hook` I guess?  That seems a bit kludgy, but I do see the risk with directory-globs.